reactioncommerce/reaction
# Fix variant view in grid product layout

This resolves #2715

### Steps to test
1. Edit a product and add images for variants
1. On the product grid, select product and choose variant view
1. Observe that images for the top three variants are displayed as below
<img width="1106" alt="screen shot 2017-08-24 at 3 44 34 pm" src="https://user-images.githubusercontent.com/25609312/29672235-8a3cc348-88e3-11e7-821b-6462fcaa9409.png">
